addons/kommander/1.1.2/kommander-1.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: kommander
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.2.0-2"
-    appversion.kubeaddons.mesosphere.io/kommander: "1.2.0-beta.0"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.1.2-1"
```

Comment:
There is a problem here: using an OLDER revision version for a newer revision may result in clusters being unable to upgrade properly, we will need to do something about this in order to proceed :thinking:

Reply:
Its just showing up like this because of the way Grace copied the existing yaml from `v1.2.0` these 2 lines and the `version` are the only difference in `1.1.1` aswell. 

TL;DR the resulting file is the same.

Reply:
I'm not sure... the way this PR is working currently, it's effectively _removing_ the latest 1.2 revision completely, is it not?

Reply:
1.2 revision should not have made its way into this branch in the first place, which is why we are removing it now - this is a `stable-1.1.x` branch used for patches on 1.1, so this branch should not contain any revisions > 1.1.x

Reply:
Ah, got it.